fifthExample.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const FifthExample = ({children}) => {
+
+    return (
+        //this code will work only when children will be an array of elements
+        //so in parent component we should put some elements which are on the same tree level
+        //between opening and closing tag of this component
+        //then elements will be wrapped in array
+        //we can also pass array of elements by children prop without using opening/closing tags approach
+        //when we provide only one element between tags or we provide something else
+        //to children props than array of elements the code will fail
+        <div>
+            {children.map((element, index) => {
```

Comment:
Actually, React provides top level api for such situations: `React.Children`: https://reactjs.org/docs/react-api.html#reactchildren

`React.Children.map` will safely traverse through children (no matter whether this is single element, array, null, undefined, etc.)

Reply:
Never heard about this API I will remember that.